examples/templates/posix_sockets.h

```diff
@@ -34,14 +34,20 @@ int open_nb_socket(const char* addr, const char* port) {
         if (sockfd == -1) continue;
 
         /* connect to server */
-        rv = connect(sockfd, servinfo->ai_addr, servinfo->ai_addrlen);
+        rv = connect(sockfd, p->ai_addr, p->ai_addrlen);
         if(rv == -1) continue;
         break;
     }  
 
     /* free servinfo */
     freeaddrinfo(servinfo);
 
+	/* did connect fail? */
+	if (!p) {
+		close(sockfd);
+		sockfd = -1;
+	}
```

Comment:
This indent has 2 extra spaced. Do you mind fixing this?

Reply:
Not sure what you mean.  Is it because I used tab indents instead of spaces ?